x-pack/plugins/apm/public/components/app/onboarding/serverless_instructions.ts

```diff
@@ -48,7 +48,7 @@ export function serverlessInstructions(
   const displayApiKeyErrorCallout = error && Boolean(errorMessage);
   const commonOptions: AgentInstructions = {
     baseUrl,
-    apmServerUrl: config.managedServiceUrl,
+    apmServerUrl: `${config.managedServiceUrl}:443`,
```

Comment:
Isn't there another way to get the port dynamically?

Reply:
@gbamparop I couldn't find a way to get that.
For stateful, the cloud plugin injects this URL and it does so by reading its config. This means the URl is set in the config in the same way.
For Serverless, the MIS URL is injected by CP team. Either we can ask them to add it at the end or hardcode it here?

Reply:
Please check with them if it's possible to add the port on their side. In the meantime we could go with this but I'd be better to not hardcode it on our side